systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -1057,4 +1059,18 @@ protected void checkReadinessLivenessProbe(String podNamePrefix, String containe
             fail("Pod with prefix " + podNamePrefix + " in name, not found");
         }
     }
+
+    protected void verifyCRDStatusCondition(Condition condition, String status, String type) {
```

Comment:
Should this be just `verifyCRStatusCondition` and `verifyCRStatusCondition`?

Reply:
yeah that's correct :)